bitnami/jaeger/templates/collector/serviceaccount.yaml

```diff
@@ -0,0 +1,21 @@
+{{- if .Values.collector.serviceAccount.create }}
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name:  {{ include "jaeger.collector.fullname" . }}
```

Comment:
We normally use a `serviceAccountName` helper so the serviceAccount.name variable is considered when creating the service account. This is useful if there are already some RBAC objects deployed

Reply:
Implemented in [c387c7c](https://github.com/bitnami/charts/pull/13480/commits/c387c7ca0efe4d88bd3f7c7f87fe5015b100a755)